instrumentation/spring/spring-boot-resources/library/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+  id("otel.library-instrumentation")
```

Comment:
Hmm, even though it actually _is_ a library instrumentation, we need the javaagent to pick this up during build (currently it doesn't).
I guess for now you could modify the `javaagent/build.gradle.kts` file and add this module to `javaagentLibs`; but maybe we should think of another way of more declarative/automatic way of including resource providers in the javaagent jar. Separate gradle plugin just for SDK plugins, like `otel.sdk-library-instrumentation`? Or maybe keeping all resources in a separate directory, and just adding this dir? @trask WDYT?

Reply:
Ok, I have opened #6534 as a follow-up issue for this.

Reply:
Thanks!